### ISSUE TITLE
PQK's outer_kernel parameter causes ValueError in sklearn routines

### ISSUE THREAD

**Issue by FredericRapp** [sQUlearn/squlearn#186]
E.g. when performing GridSearchCV, the outer_kernel parameter is not accessed by the get_ / set_params of the high-level methods:

```
projected_kernel = ProjectedQuantumKernel(
    encoding_circuit=encoding_circuit, executor=executor, parameter_seed=0
)
qkrr = QKRR(quantum_kernel=projected_kernel)
param_grid = {
        "outer_kernel": ["Gaussian", "DotProduct"],
        "num_qubits": [2, 3, 4],
        "num_layers": [1, 2, 3],
        "alpha": [0.01, 0.1, 1.0]
}
grid_search = GridSearchCV(qkrr, param_grid, cv=5, n_jobs=-1, verbose=1)

grid_search.fit(X, y)
```

This example leads to: 
```
ValueError: Invalid parameter 'outer_kernel'. Valid parameters are ['alpha', ...

**Comment by David-Kreplin** [comment]
Is fixed, PR will come soon.

**Comment by David-Kreplin** [comment]
Fixed by PR #192
